lib/ui/semantics.dart

```diff
@@ -307,6 +309,40 @@ class SemanticsFlag {
   /// is a password or contains other sensitive information.
   static const SemanticsFlag isObscured = const SemanticsFlag._(_kIsObscuredIndex);
 
+  /// Whether the semantics node is the root of a subtree for which values
+  /// should be announced.
+  /// 
+  /// When a node with this flag is removed from the semantics tree, the 
+  /// framework will select the last in depth-first, paint order node with this
+  /// flag.  When a node with this flag is added to the semantics tree, it is
+  /// selected automatically, unless there were multiple nodes with this flag
+  /// added.  In this case, the last added node in depth-first, paint order
+  /// will be selected.
+  /// 
+  /// From this selected node, the framework will search in depth-first, paint
+  /// order for the first node with a [namesRoute] flag and a non-null, 
+  /// non-empty label. The [namesRoute] and [scopesRoute] flags may be on the 
+  /// same node. The label of the found node will be announced as an edge 
+  /// transition. if no non-empty, non-null label is found then:
+  ///   
+  ///   * VoiceOver will make a chime announcement.
+  ///   * TalkBack will make no announcement
+  /// 
+  /// This is used in widgets such as Routes, Drawers, and Dialogs to 
+  /// communicate significant changes in the visible screen.
+  static const SemanticsFlag scopesRoute = const SemanticsFlag._(_kScopesRouteIndex);
+
+  /// Whether the semantics node label is the name of a visually distinct
+  /// route.
+  /// 
+  /// This is used by certain widgets like Drawers and Dialogs, to indicate
+  /// that the node's semantic label can be used to announce an edge triggered
+  /// semantics update.
+  /// 
+  /// Updating this label within the same active route subtree will not cause 
+  /// additional semantic label updates.
+  static const SemanticsFlag namesRoute = const SemanticsFlag._(_kNamesRouteIndex);
```

Comment:
Should this document that nodes with this flag are still going to receive a11y focus?